packages/plugins/rudder_plugin_interface/lib/src/models/web_config_interfaces/load_opts.dart

```diff
@@ -0,0 +1,30 @@
+typedef OnLoadedCallback = void Function(dynamic analytics);
+
+enum EventsTransportMode { xhr, beacon }
```

Comment:
You can skip this. It is not a valid load API option at this point.

Reply:
Done